src/pubsub/index.js

```diff
@@ -586,13 +611,11 @@ class PubsubBaseProtocol extends EventEmitter {
     const from = this.peerId.toB58String()
     let msgObject = {
       receivedFrom: from,
-      from: from,
       data: message,
-      seqno: utils.randomSeqno(),
       topicIDs: [topic]
     }
 
-    // ensure that any operations performed on the message will include the signature
+    // ensure that any operations performed on the message will include the signature, if it should exist
```

Comment:
```suggestion
    // ensure that the message follows the signature policy
```
Seems cleaner this way